src/useCases/getUserByEmailAddress.ts

```diff
@@ -2,7 +2,8 @@ import Database from "types/Database"
 import PromiseResult from "types/PromiseResult"
 import User from "types/User"
 
-export default (db: Database, emailAddress: string): PromiseResult<User | null> => {
+/* eslint-disable require-await */
+export default async (db: Database, emailAddress: string): PromiseResult<User | null> => {
```

Comment:
ts-server was complaining that this should be an async function but eslint would remove the await  in the return and then complain the function had no await.